MetadataProcessor.Shared/Tables/nanoGenericParamTable.cs

```diff
@@ -82,7 +139,49 @@ protected override void WriteSingleItem(
                 return;
             }
 
-            // TODO
+            var writerStartPosition = writer.BaseStream.Position;
+
+            // number
+            writer.WriteUInt16((ushort)item.Position);
+
+            // flags
+            writer.WriteUInt16((ushort)item.Attributes);
+
+            // find owner
+            if (item.Owner is TypeDefinition &&
+                _context.TypeDefinitionTable.TryGetTypeReferenceId(item.Owner as TypeDefinition, out ushort referenceId))
+            {
+                // is TypeDef
```

Comment:
I saw further (line 143 nanoMemberReferenceTable) that MethodDef and MethodRef is not supported but here the comments says TypeDef. Is this a miss comment? And maybe then add a note here as well

Reply:
Our metadata doesn't have support for ModuleRef and MethodDef. This one here is about finding if the owner is a TypeDefinition. The comment here (just like the one in the following ELSE IF) is there to stress what this IF is for as it may not be that obvious because the result comes in the out parameter.